app/views/documents/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= content_for :back_link, document_path(@document) %>
-<% content_for :title, @document.title || "Untitled document" %>
+<% content_for :title, @document.title || ("Create " + @document.document_type).humanize %>
```

Comment:
Rather than needing to use humanize here on document.document_type you could use `Document.first.document_type_schema.name` which will give you it in sentence case.

We may still hit issues as to whether these are all considered proper nouns or not, but hopefully https://trello.com/c/7XfrouK7/90-allow-content-designers-to-edit-the-application-microcopy-m may offer some micro control if needed.